src/account.ts

```diff
@@ -115,7 +118,10 @@ export const generateAddress2 = function(
  * Returns true if the supplied address belongs to a precompiled account (Byzantium).
  */
 export const isPrecompiled = function(address: Buffer | string): boolean {
-  const a = unpad(address)
+  if (typeof address === 'string') {
+    enforceHexString(address)
+  }
+  const a = ethjsUtil.stripHexPrefix(unpad(address))
   return a.length === 1 && a[0] >= 1 && a[0] <= 8
 }
```

Comment:
Just for reference wrt the safety of this change. Complete [stripHexPrefix][1] method is:
```javascript
module.exports = function stripHexPrefix(str) {
  if (typeof str !== 'string') {
    return str;
  }

  return isHexPrefixed(str) ? str.slice(2) : str;
}
```
[1]: https://github.com/SilentCicero/strip-hex-prefix/blob/master/src/index.js

Reply:
This is irrelevant.